jp.yvt.OpenSpades.json

```diff
@@ -52,7 +52,7 @@
         {
           "type": "archive",
           "url": "https://imagemagick.org/download/ImageMagick.tar.gz",
```

Comment:
You definitely want to change the URL to a specific version, too. If you don't do that, the build will fail in the future when imagemagick releases a new version.

There is directory of releases here: https://imagemagick.org/download/releases/

Reply:
Thanks for the heads up! I will definitely do that.